spec/specHelper.js

```diff
@@ -0,0 +1,60 @@
+/* jshint node: true */
+/* jshint esnext: true */
+/* global jasmine */
+'use strict';
+
+let ParticipantBuilder = function() {
```

Comment:
Could you extract the whole participantBuilder stuff to another pull request, then I could merge this straight away.